packages/web3torrent/src/components/form/form-button/FormButton.test.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement } from "react";
+import ReactDOM from "react-dom";
+
+import { FormButton } from "./FormButton";
+
+describe("<FormButton />", () => {
+  let container: HTMLDivElement;
+  let button: HTMLButtonElement;
+
+  const render = (element: ReactElement) => {
+    ReactDOM.render(element, container);
+    button = container.querySelector("button")!;
+  };
+
+  beforeEach(() => {
+    container = document.createElement("div");
+  });
+
+  it("renders without crashing", () => {
+    render(<FormButton />);
+  });
+
+  it("renders a button with default properties", () => {
+    render(<FormButton />);
+    expect(button.disabled).toBe(false);
+    expect(button.className).toBe("button");
+    expect(button.type).toBe("button");
+  });
+
+  it("renders a disabled button", () => {
+    render(<FormButton disabled={true} />);
+    expect(button.disabled).toBe(true);
+  });
+
+  it("renders a button with a custom class", () => {
+    const className = "foo";
+    render(<FormButton className={className} />);
+    expect(button.className).toBe(className);
+  });
+
+  it("renders text content inside a button", () => {
+    const content = "Click me";
+    render(<FormButton>{content}</FormButton>);
+    expect(button.innerHTML).toBe(content);
+  });
+
+  it("renders elements inside a button", () => {
+    const content = (
+      <label>
+        <span>❓</span> Help
```

Comment:
This is probably triggering an a11y warning. You can fix it with `role="img" aria-label="Question mark"`.

Reply:
but... it's a test! (I'll add it, thanks for solving it for me xD )